compiler/resolution/resolveFunction.cpp

```diff
@@ -1335,7 +1335,7 @@ static void addLocalCopiesAndWritebacks(FnSymbol*  fn,
            //  destructor calls delete on any fields.  I think we
            //  probably need a similar change in the INOUT/IN case
            //  above.  See test/types/records/sungeun/destructor3.chpl
-           //  and test/users/recordbug3.chpl.
+           //  and test/users/ferguson/recordbug3.chpl.
```

Comment:
I think this comment is referring to these tests as futures.  But both have been retired.

recordbug3 by 70d5ea40, and destructor3 by 5f19b38.

Reply:
Something is certainly out of date here, but I'm not sure to what extent it requires updating...  Maybe @noakesmichael has a better idea?

Reply:
Let's follow @mppf's suggestion of keeping this PR critical-thinking-free™, and just leave the rest of the comment as is. I'll still point this comment out to Mike to see how it should be updated separately.